tools/tests/com.liferay.ide.project.core.tests/src/com/liferay/ide/project/core/tests/NewLiferayPluginProjectPortletNameOpTests.java

```diff
@@ -69,11 +69,11 @@ public static void removePluginsSDK() throws CoreException {
 			if (project != null && project.isAccessible() && project.exists()) {
 				try {
 					project.close(new NullProgressMonitor());
-					project.delete(true, true, new NullProgressMonitor());
+					project.delete(true, new NullProgressMonitor());
```

Comment:
It will only delete resource from Eclipse not from the disk. I try a lot of time in my machine, it can avoid setupSDK problem. In Jenkins test, it will show the sdk location is not valid sometimes instead of deleting resource error. It's because we catch the delete resource error and not throw it. When we run a new test, it will use the old sdk which is not deleted entirely. So it will fail the test for sdk location is not valid.

Reply:
thanks for explaination